doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -257,6 +257,30 @@ ifdef::PAM[]
 
 == Process designer
 
+.Validation for a signal name fails if the name includes spaces or colons (`:`) [https://issues.redhat.com/browse/RHPAM-2557[RHPAM-2557]]
+
+Issue: In the process designer, the validation for a signal name fails if the name includes spaces, colons (`:`), or other special characters.
+
+Steps to reproduce:
+
+. Open a BPMN project in the process designer.
+. Add a `Signal` node on the canvas.
```

Comment:
I would either make this l/c no formatting or if you are quoting the UI, then as it is in the UI and bolded. Even if Signal isi in the UI, I would use just the plain work here. Just as if I were talking about a gateway, I would "Add a gateway to the canvas."